src/graph_engine.jl

```diff
@@ -1182,6 +1176,8 @@ struct AnonymousVariable{M, C}
     context::C
 end
 
+Base.broadcastable(v::AnonymousVariable) = Ref(v)
```

Comment:
Lets define `broadcastable` for other types as well. As I understand currently we rely on a `length` machinery.